app/views/state_file/state_file_pages/about_page.html.erb

```diff
@@ -19,9 +19,9 @@
 <section>
   <%= link_to "Start Test AZ", Navigation::StateFileAzQuestionNavigation::FLOW.first.to_path_helper(us_state: "az"), class: "button" %>
   <%= link_to "Start Test NY", Navigation::StateFileNyQuestionNavigation::FLOW.first.to_path_helper(us_state: "ny"), class: "button" %>
-  <% if session[:state_file_intake] %>
+  <% if current_intake %>
     <div class="with-padding-med">
-      Your session has this intake: <b><%= session[:state_file_intake] %></b>
+      Your session has this intake: <b><%=current_intake.state_code %> <%= current_intake.id %></b>
```

Comment:
I figured the state code was useful here too - to make sure the intake is the one we expect